util/migration-rocksdb/src/lib.rs

```diff
@@ -161,6 +161,42 @@ impl Migration for ChangeColumns {
 	}
 }
 
+pub struct VacuumAccountsBloom {
+	pub column_to_vacuum: u32,
+	pub columns: u32,
+	pub version: u32,
+}
+
+impl Migration for VacuumAccountsBloom {
+	fn pre_columns(&self) -> u32 { self.columns }
+	fn columns(&self) -> u32 { self.columns }
+	fn alters_existing(&self) -> bool { true }
+	fn version(&self) -> u32 { self.version }
+
+	fn migrate(&mut self, db: Arc<Database>, _config: &Config, _dest: &mut Database, col: u32) -> io::Result<()> {
+		if col != self.column_to_vacuum {
+			return Ok(())
+		}
+		let num_keys = db.num_keys(COL_ACCOUNT_BLOOM)?;
+		info!(target: "migration", "Removing accounts existence bloom ({} keys)", num_keys + 1);
+		let mut batch = DBTransaction::with_capacity(num_keys as usize);
+		const COL_ACCOUNT_BLOOM: u32 = 5;
+		const ACCOUNT_BLOOM_HASHCOUNT_KEY: &'static [u8] = b"account_hash_count";
+		for (n, (k,_)) in db.iter(COL_ACCOUNT_BLOOM).enumerate() {
+			batch.delete(COL_ACCOUNT_BLOOM, &k);
+			if n > 0 && n % 10_000 == 0 {
+				info!(target: "migration", "  Account Bloom entries queued for deletion: {}", n);
+			}
+		}
+		batch.delete(COL_ACCOUNT_BLOOM, ACCOUNT_BLOOM_HASHCOUNT_KEY);
```

Comment:
we could use the new (not published yet) kvdb api to simplify this https://github.com/paritytech/parity-common/pull/368

Reply:
This migration is really quick to run. We can tweak this when we get around to updating the `kvdb-*`s.